tests/stdlib/vector.c3

```diff
@@ -2,6 +2,10 @@
 @require_once "std/vector.c3"
 @require_once "std/format.c3"
 
+@operator[T] [] : (self : Vector<T>&, index : isize) -> T mut& = {
```

Comment:
This is so nice that it should be in `std/vector.c3`

Reply:
Yup, this'll break bootstrap tho. We need to wait a commit

Reply:
Ah, of course